File: atcoder.jp/contests/abc111/tasks/abc111_b/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const divCeil = (a: number, b: number) => Math.trunc((a + b - 1) / b);

const main = () => {
  const N = +input();

  const ans = divCeil(N, 111) * 111;
  console.log(ans);
};

main();

File: atcoder.jp/contests/abc111/tasks/abc111_b/main.cpp
#include <bits/stdc++.h>
using namespace std;

int div_ceil(int a, int b) { return (a + b - 1) / b; }

int main() {
  int N;
  cin >> N;

  int ans = div_ceil(N, 111) * 111;
  cout << ans << endl;
}
